routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
+
+    attr_reader :subscriber, :callback, :topics, :events
+
+    def initialize(options)
+      @subscriber = options.fetch('subscriber')
+      @callback   = options.fetch('callback')
+      @topics     = options.fetch('topics')
+      @events     = options.fetch('events').symbolize_keys
```

Comment:
`symbolize_keys` doesn't exist in vanialla Ruby, so you're probably missing a `require` at the top of this file?